showcase/src/app/demo-ui/pages/minimap/demo-minimap-page.component.ts

```diff
@@ -1,12 +1,13 @@
-import { Component, Inject, OnInit } from "@angular/core";
+import { Component, Inject, OnInit, ViewEncapsulation } from "@angular/core";
 import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
 import { ReferenceLink } from "../../../shared/components";
 import { ItemVisibility, StarkMinimapItemProperties } from "@nationalbankbelgium/stark-ui";
 
 @Component({
 	selector: "demo-minimap",
 	templateUrl: "./demo-minimap-page.component.html",
-	styleUrls: ["./demo-minimap-page.component.scss"]
+	styleUrls: ["./demo-minimap-page.component.scss"],
+	encapsulation: ViewEncapsulation.None // Important
```

Comment:
Why do we need `ViewEncapsulation.None` here ?

Reply:
To apply the style to the child component. But after I move the fix to stark-ui it won't be needed anymore.